pkg/decision/composite_experiment_service.go

```diff
@@ -34,14 +34,16 @@ type CompositeExperimentService struct {
 
 // NewCompositeExperimentService creates a new instance of the CompositeExperimentService
 func NewCompositeExperimentService() *CompositeExperimentService {
+	persistingExperimentService := NewPersistingExperimentService(NewExperimentBucketerService())
```

Comment:
Should we only apply the persisting wrapper when they use an option func and actually provide a user profile service? Otherwise, most of the time, this persisting experiment service is running, but not doing anything interesting.

Reply:
Yeah you are right. Let me fix that.